pkg/cache/v3/simple.go

```diff
@@ -210,6 +211,30 @@ func (cache *snapshotCache) SetSnapshot(node string, snapshot Snapshot) error {
 				delete(info.watches, id)
 			}
 		}
+
+		// We only calculate version hashes when using delta. We don't want to do this when using SOTW.
```

Comment:
note that we avoid this due to computational cost of calculating hashes

Reply:
👍 Will provide a note there